app/controllers/site_controller.rb

```diff
@@ -56,34 +56,51 @@ def load_global
     @sdgs = YAML.load(File.open("#{Rails.root}/lib/data/content/sdgs.yml", 'r'))
   end
 
+  def sum_country_areas(total_area_by_country)
+    country_total_area = {}
+    total_area_by_country.flatten.each do |country_data|
+      next if country_data["iso3"].include? "/" #remove areas which have multiple iso
+      next if country_data["iso3"].include? "ABNJ" #remove ABNJ
+      country_total_area[country_data["iso3"]] = country_total_area[country_data["iso3"]].nil? ?  country_data["sum"] : country_total_area[country_data["iso3"]] + country_data["sum"]
```

Comment:
Perhaps you could simply this by first initializing the hash when empty and then just adding the sum, like:

```
country_total_area[country_data["iso3"]] ||= 0
country_total_area[country_data["iso3"]] += country_data["sum"]
```

Worth checking the correctness but it should be ok, and also easier to read I believe ;)

Reply:
Many thanks I will try this :)